src/Configuration/features/atlas/config.yml

```diff
@@ -122,6 +122,7 @@ actions:
   - !scheduledTask: {path: '\Microsoft\Windows\InstallService\ScanForUpdates', operation: delete}
   - !scheduledTask: {path: '\Microsoft\Windows\InstallService\ScanForUpdatesAsUser', operation: delete}
   - !scheduledTask: {path: '\Microsoft\Windows\InstallService\SmartRetry', operation: delete}
+  - !scheduledTask: {path: '\Microsoft\Windows\Maintenance\WinSAT', operation: delete}
```

Comment:
I don't think that WinSAT is automatically ran on modern Windows, no point in deleting it, although the scheduled tasks need to be redone in general.

Reply:
It is only disabled if it tends to enable itself automatically, though it should be prevented before.

Reply:
It's fine for now, it can be kept.